packages/manager/apps/public-cloud/src/sidebar/sidebar.less

```diff
@@ -0,0 +1,143 @@
+@background-color: rgb(244,248,251);
```

Comment:
Just a quick note (I noticed that is still a work in progress) but some colors are used multiple times:)